lib/pilot/build_manager.rb

```diff
@@ -7,9 +7,10 @@ def upload(options)
 
       Helper.log.info "Ready to upload new build to TestFlight (App: #{app.apple_id})...".green
 
-      package_path = PackageBuilder.new.generate(apple_id: app.apple_id,
-                                                 ipa_path: config[:ipa],
-                                             package_path: "/tmp")
+      package_path = FastlaneCore::IpaUploadPackageBuilder.new.generate(app_id: app.apple_id,
+                                                                      ipa_path: config[:ipa],
+                                                                  package_path: "/tmp",
+                                                                      platform: "appletvos")
```

Comment:
Is the tvOS binary always separate from the iOS one? How is the binary different? If we can fetch it from the plist we should be able to do that:
```
FastlaneCore::IpaFileAnalyser.fetch_info_plist_file
```